### DISCUSSION TITLE
IconTintColorBehavior for windows

### DISCUSSION THREAD

**Discussion by kperdlich** [CommunityToolkit/Maui#1180] [General]
Is anyone interested in a windows support for the IconTintColorBehavior? 😃 
I opened a draft with my current implementation: https://github.com/CommunityToolkit/Maui/pull/1179

![image](https://github.com/CommunityToolkit/Maui/assets/6242549/4404b5ff-da29-4f04-aa0c-2005dcee0af7)

**Comment by pictos** [comment]
@kperdlich can you please open an issue, that way we can track it better
